app/types.go

```diff
@@ -0,0 +1,39 @@
+package app
```

Comment:
nit: I don't love the types.go filename. Alternate suggestion: `cosmos_app.go`

Reply:
swapped :^)